pkg/bytecode/vm.go

```diff
@@ -160,9 +160,7 @@ func (vm *VM) Run() error {
 				key := vm.popStringVal()
 				m[string(key)] = val
 			}
-			if err := vm.push(m); err != nil {
-				return err
-			}
+			err = vm.push(m)
```

Comment:
reducing cyclomatic complexity of `vm.Run()` - this will need a rethink and a fixup PR very soon as it is at 30/30 branches

Reply:
Just put in //golint:ignore whatever line. The `Run()` method is special in that it is written to be optimal and will likely need to break the rules to do so.